components/screens/BlogScreen/BlogStyleSheet.js

```diff
@@ -0,0 +1,45 @@
+import {StyleSheet} from 'react-native';
```

Comment:
Can we rename this file to `BlogStyles.js`? _(We changed the naming convention on you)._